src/Actions/index.test.js

```diff
@@ -86,4 +86,26 @@ describe('actions', () => {
     const result = actions.addPaletteToProject(mockProjectId, paletteToSave);
     expect(result).toEqual(expected);
   });
+
+  it('should return an objet with DELETE_PALETTE and a palette id', () => {
+    const mockPaletteId = 4;
+    const mockProjectId = 3;
+    const expected = {
+      type: 'DELETE_PALETTE',
+      paletteId: mockPaletteId,
+      projectId: mockProjectId
+    };
```

Comment:
As per our discussion, I think that passing in the project id and the palette id is a great way to do this. Nice work!